src/wallet/rpcwallet.cpp

```diff
@@ -2948,6 +2948,7 @@ UniValue setbip69enabled(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+#if ENABLE_MINER
```

Comment:
would prefer to wrap `#include "rpc/mining.h"` too https://github.com/UdjinM6/dash/commit/3567fdc79d41c6dab3c75aa7ece2d5a7a8543d34#diff-df7d84ff2f53fcb2a0dc15a3a51e55ceR17

Reply:
https://github.com/dashpay/dash/pull/3043/commits/42cf74dec48e81a528b181e199a13e241aa2aacc